dace/codegen/instrumentation/papi.py

```diff
@@ -736,5 +736,7 @@ def accumulate_byte_movement(outermost_node, node, dfg: StateGraphView, sdfg, st
             elif isinstance(node, Tasklet):
                 return itcount * symbolic.pystr_to_symbolic(
                     PAPIUtils.get_tasklet_byte_accesses(node, dfg, sdfg, state_id))
+            elif isinstance(node, nodes.AccessNode):
+                return 0
```

Comment:
Seems to be introduced by fusion of maps. Only had a quick look at the method, I can try to figure it out or do you know what is supposed to happen in that case? @tbennun